src/rhsm/connection.py

```diff
@@ -775,9 +775,12 @@ def _load_manager_capabilities(self):
         """
         self.capabilities = {}
         self.capabilities = self.conn.request_get("/status/")
-        self.capabilities = self.capabilities['managerCapabilities']
-        log.debug("Server has the following capabilities: %s",
-                  self.capabilities)
+        try:
+            self.capabilities = self.capabilities['managerCapabilities']
+            log.debug("Server has the following capabilities: %s",
+                    self.capabilities)
+        except KeyError as e:
+            log.debug("Unable to retrieve managerCapabilities: %s" % str(e))
```

Comment:
```
log.debug(""Unable to retrieve managerCapabilities: %s" , e)
```
That avoids evaluating e until the logger is prepared to emit the message (ie, if logs are set at INFO, str/repr of e doesn't need to happen and is skipped).